packages/components/molecules/f-card-with-content/package.json

```diff
@@ -54,8 +54,8 @@
     "@justeat/f-card": "3.x"
   },
   "devDependencies": {
-    "@justeat/f-button": "4.0.0",
-    "@justeat/f-card": "4.0.0",
+    "@justeat/f-button": "3.x",
```

Comment:
Going to `3.x` for devDeps because that's what the peerDeps specify. @ashleynolan Would you prefer I update the peerDeps to `4.x`? If so, shall I make this a major version bump instead?

Reply:
Yes  please 👍